### DISCUSSION TITLE
frigate viewer android app

### DISCUSSION THREAD

**Discussion by zajc3w** [blakeblackshear/frigate#14504] [Ask A Question]
Is this app supposed to work over HTTPS from remote location?
I have frigate setup in docker for testing, coral TPU incoming so only one 720p camera configured for now.
it's behind traefok reverse proxy and i can use web interface using my domain.
but frigate viewer app keeps throwing incorrect credentials used. I even added used with simple pass, so i couldn't make a typo.
frigate container logs show: `2024-10-22 10:20:12.609252382  172.18.0.2 - admin [22/Oct/2024:10:20:12 +0000] "GET /api/config HTTP/1.1" 401 179 "-" "okhttp/4.9.2" "5.15
 `
even if I open port 5000 directly to the net i get the same errors

sorry if frigate viewer is not frigate team app

**Comment by NickM-27** [comment]
We have nothing to do with the Frigate viewer app 

https://docs.frigate.video/configuration/pwa is the recommended way to use Frigate as an app

**Comment by hawkeye217** [comment]
The app you're referring to is not something we produce or maintain.

**Comment by zajc3w** [comment]
Just realised that, and I found app repo on GitHub.
It is an app error not supporting Https authentication...already existing
issue in its repo
